Plugins not added when specified by rule converters

<!--
👋 Hi, thanks for filing an issue on tslint-to-eslint-config! 💖
Please fill out all fields below to ensure your issue is addressed.

If your issue doesn't provide enough info to fully explain or reproduce your bug, it will be closed. 😦
-->

### 🐛 Bug Report

-   `tslint-to-eslint-config` version:  1.0.0
-   ESLint version: 7.2.0
-   Node version: 14.4.0

#### Actual Behavior

after converting, show serveral errors:
```
    1:1   error  Definition for rule '@typescript-eslint/class-name-casing' was not found      @typescript-eslint/class-name-casing
    1:1   error  Definition for rule '@typescript-eslint/interface-name-prefix' was not found  @typescript-eslint/interface-name-prefix
    1:1   error  Definition for rule 'import/order' was not found                              import/order
    1:1   error  Definition for rule 'jsdoc/check-alignment' was not found                     jsdoc/check-alignment
    1:1   error  Definition for rule 'jsdoc/check-indentation' was not found                   jsdoc/check-indentation
    1:1   error  Definition for rule 'jsdoc/newline-after-description' was not found           jsdoc/newline-after-description
    1:1   error  Definition for rule 'prefer-arrow/prefer-arrow-functions' was not found       prefer-arrow/prefer-arrow-functions
```

#### Expected Behavior

No Error 
#### Reproduction

Just following up, I re-ran with the posted tslint settings and got this message:

```
✨ 43 rules replaced with their ESLint equivalents. ✨

❗ 5 ESLint rules behave differently from their TSLint counterparts ❗
  Check ./tslint-to-eslint-config.log for details.

❓ 2 rules are not known by tslint-to-eslint-config to have ESLint equivalents. ❓
  The "@typescript-eslint/tslint/config" section of ./.eslintrc.js configures eslint-plugin-tslint to run them in TSLint within ESLint.
  Check ./tslint-to-eslint-config.log for details.

☠ Prettier plugins are missing from your configuration. ☠
  We highly recommend running tslint-to-eslint-config --prettier to disable formatting ESLint rules.
  See https://github.com/typescript-eslint/tslint-to-eslint-config/blob/master/docs/FAQs.md#should-i-use-prettier.

♻ Consider using --comments to replace TSLint comment directives in your source files. ♻

⚡ 7 new packages are required for this ESLint configuration. ⚡
  npm install @typescript-eslint/eslint-plugin @typescript-eslint/eslint-plugin-tslint @typescript-eslint/parser eslint eslint-plugin-import eslint-plugin-jsdoc eslint-plugin-prefer-arrow --save-dev

✅ All is well! ✅
```

After installing those packages, running ESLint gives these complaints:

```
  1:1   error  Definition for rule 'import/order' was not found                         import/order
  1:1   error  Definition for rule 'jsdoc/check-alignment' was not found                jsdoc/check-alignment
  1:1   error  Definition for rule 'jsdoc/check-indentation' was not found              jsdoc/check-indentation
  1:1   error  Definition for rule 'jsdoc/newline-after-description' was not found      jsdoc/newline-after-description
  1:1   error  Definition for rule 'prefer-arrow/prefer-arrow-functions' was not found  prefer-arrow/prefer-arrow-functions       
```

So, yes, even with all our recent improvements, there are still some issues in the rule converters, it seems!

--- a/src/creation/writeConversionResults.test.ts
+++ b/src/creation/writeConversionResults.test.ts
@@ -60,6 +60,7 @@ describe("writeConversionResults", () => {
                     ruleSeverity: "error",
                 },
             ],
+            plugins: new Set(["eslint-plugin-example"]),
         });
         const fileSystem = { writeFile: jest.fn().mockReturnValue(Promise.resolve()) };
 
@@ -85,7 +86,11 @@ describe("writeConversionResults", () => {
                     project: "tsconfig.json",
                     sourceType: "module",
                 },
-                plugins: ["@typescript-eslint", "@typescript-eslint/tslint"],
+                plugins: [
+                    "eslint-plugin-example",
+                    "@typescript-eslint",
+                    "@typescript-eslint/tslint",
+                ],
                 rules: {
                     "@typescript-eslint/tslint/config": [
                         "error",

--- a/src/creation/writeConversionResults.ts
+++ b/src/creation/writeConversionResults.ts
@@ -16,11 +16,11 @@ export const writeConversionResults = async (
     summarizedResults: SummarizedResultsConfiguration,
     originalConfigurations: AllOriginalConfigurations,
 ) => {
-    const plugins = ["@typescript-eslint"];
+    const plugins = new Set([...summarizedResults.plugins, "@typescript-eslint"]);
     const { eslint, tslint } = originalConfigurations;
 
     if (summarizedResults.missing.length !== 0) {
-        plugins.push("@typescript-eslint/tslint");
+        plugins.add("@typescript-eslint/tslint");
     }
 
     const output = removeEmptyMembers({
@@ -33,7 +33,7 @@ export const writeConversionResults = async (
             project: "tsconfig.json",
             sourceType: "module",
         },
-        plugins,
+        plugins: Array.from(plugins),
         rules: formatConvertedRules(summarizedResults, tslint.full),
     });
 

